# GPS information
# time 2020/09/23
# added by Xu Liu
# https://pixhawk.ethz.ch/mavlink/#ATTITUDE

std_msgs/Header header

uint8 fix_type
uint8 satellites_num